pmd-java/etc/grammar/Java.jjt

```diff
@@ -1854,7 +1854,7 @@ void ConstructorDeclaration(int modifiers) :
 Token t;}
 {
     [ TypeParameters() ]
-  <IDENTIFIER> FormalParameters() [ "throws" NameList() ]
+  t=<IDENTIFIER> {jjtThis.setImage(t.image);} FormalParameters() [ "throws" NameList() ]
```

Comment:
In fact the last token matched can be accessed through getToken(0) so no need for a Token variable. `jjtThis.setImage(getToken(0).getImage());`

Reply:
@oowekyala Thanks . Implemented the same.